app/services/support_interface/referee_survey_export.rb

```diff
@@ -0,0 +1,30 @@
+module SupportInterface
+  class RefereeSurveyExport
+    def self.call
+      references = ApplicationReference.where.not(questionnaire: nil)
+      references_with_feedback = references.reject do |reference|
+        reference.questionnaire.values.all? { |response| response == ' | ' }
```

Comment:
This is ugly. At the moment if a referee does not give any feedback. The questionnaire attribute is updated to ' | ' for each question due to this 

![image](https://user-images.githubusercontent.com/42515961/75174460-85018880-5728-11ea-9a46-8ab4eb2f8fef.png)


I'm not sure whether changing this code so if they do not respond the questionnaire attributes value remains nil, then running a data migration to fix the current data is preferable.

Reply:
It also will be a lot less taxing on the db as we scale. Not sure if I'm overthinking something that will be sparsely used though.

Reply:
Will look at this with you tomorrow 👍